libs/ui/src/seal.tsx

```diff
@@ -38,7 +38,9 @@ export function Seal({
   maxWidth,
   style,
   inverse,
-}: SealProps): JSX.Element {
+}: SealProps): JSX.Element | null {
```

Comment:
Are we sure a Seal is only used in this component?

Reply:
Good question - I just searched through the repo and the answer is yes!